### PR TITLE
fix picker multi value

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Fix bug with picker, now the single picker value is controlled by the user (with the value prop) and the multi picker is controlled by the user (with the value prop) and controlled by the component (with multiDraftValue state and multiFinalValue)
added TODO task to add an initialValue prop in the future to make it more clear to the user in multi mode.

Fix https://github.com/wix/react-native-ui-lib/issues/1615

## Changelog
Picker - fix multi mode value

### CODE REVIEW

File: src/components/picker/index.js [hunk 1]
```diff
@@ -372,15 +357,15 @@ class Picker extends Component {
...
+    const {showExpandableModal, selectedItemPosition, multiDraftValue} = this.state;
 
     if (renderCustomModal) {
       const modalProps = {
         visible: showExpandableModal,
         toggleModal: this.toggleExpandableModal,
         onSearchChange: this.onSearchChange,
         children,
-        onDone: () => this.onDoneSelecting(value),
+        onDone: () => this.onDoneSelecting(multiDraftValue),
```
[the remark this answers is not in the document]
Comment (ethanshar) [reply src/components/picker/index.js:368]: What do you mean by "all types"? 
This state value is not used in single mode, therefore its name is `multiDraftValue`

[the remark this answers is not in the document]
Comment (M-i-k-e-l) [reply src/components/picker/index.js:368]: That's what I've written, but will a **future** developer know that? - I feel the code is (still) unclear about the fact it's only for `MULTI`.

[the remark this answers is not in the document]
Comment (ethanshar) [reply src/components/picker/index.js:368]: What do you suggest?

[the remark this answers is not in the document]
Comment (M-i-k-e-l) [reply src/components/picker/index.js:368]: I don't have a simple solution, maybe adding a comment for now.

**Review by M-i-k-e-l — APPROVED** [review]

File: src/components/picker/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (M-i-k-e-l) [reply src/components/picker/index.js:368]: Approved, merge with\without adding a comment.

[the remark this answers is not in the document]
Comment (ethanshar) [reply src/components/picker/index.js:368]: @lidord-wix 
I agree this specific use case worth a comment. 
Let's add a comment and merge

**Pull request merged**